### PR TITLE
Add Perl 6 / Raku resources

### PR DESCRIPTION
**Pull request by uzluisf**
## What does this PR do?
Add Resource(s) 
Add info

### How do we know it's really free?
There are links to authors' pages.
### Checklist:
- [x] Not a duplicate
- [x] Included author(s) if appropriate
- [x] Lists are in alphabetical order
- [x] Needed indications added (PDF, access notes, under construction)

### CODE REVIEW

**Comment by uzluisf** [comment]
I think it should be fine now ;)!

**Comment by bluetata** [comment]
@uzluisf   If you add a blank line on line **288** in the free-programming-books-es.md file, the machine checking will be pass.

File: free-programming-books-es.md [hunk 1]
```diff
@@ -285,6 +286,10 @@
 * [Tutorial Perl](http://kataix.umag.cl/~ruribe/Utilidades/Tutorial%20de%20Perl.pdf) (PDF)
 * [Tutoriales de Perl](http://perlenespanol.com/tutoriales/) (HTML)
```
[the remark this answers is not in the document]
Comment (uzluisf) [reply free-programming-books-es.md]: I've added it. Thanks!

**Comment by eshellman** [comment]
Sorry to leave this hanging. thanks for the help @bluetata, welcome to free-programming-books, @uzluisf !

**Pull request merged**